app/models/sample.rb

```diff
@@ -67,7 +67,7 @@ class Sample < ApplicationRecord
   belongs_to :user, optional: true, counter_cache: true # use .size for cache, use .count to force COUNT query
   belongs_to :host_genome, optional: true
   has_many :pipeline_runs, -> { order(created_at: :desc) }, dependent: :destroy
-  has_and_belongs_to_many :backgrounds, through: :pipeline_runs
+  has_many :backgrounds, through: :pipeline_runs
```

Comment:
Same comment about many-to-many relationship. I think this change is wrong and unnecessary